### ISSUE TITLE
the `enable: false` fails to work if there is a comment on the line before it

### ISSUE THREAD

**Issue by gimantha** [ballerina-platform/ballerina-lang#34488]
Consider the following code. 
```ballerina
import ballerina/test;
@test:Config {
    //comment
    enable: false
}
public function testMe() {
    test:assertEquals("E", "E");
}
```

I have set `enable` as `false`. but for some reason, if I put a comment before that line inside the annotation, test will run.

SL-beta 6

**Comment by lochana-chathura** [comment]
There are a few other scenarios. 
[1].
```bal
@ test:Config{ // Here the whole annotation gets ignored due to the space/newline after @ token
    enable:false
}
function testDisableFunc1() {
    test:assertTrue(false, msg = "this test is not expected to run");
}
```
[2].
```bal
@test:Config{
    "enable":false // Here field name was not recognized because it is a string literal
}
function testDisableFunc1() {
    test:assertTrue(false, msg = "this test is not expected to run");
}
```

**Comment by lochana-chathura** [comment]
The reason for these edge cases is from the usage of `toString()` instead of `text()` in the testerina TestProccesor.

**Comment by lochana-chathura** [comment]
@Dilhasha I already fixed it locally. I will send a PR. :)

**Comment by Dilhasha** [comment]
> @Dilhasha I already fixed it locally. I will send a PR. :)

Thanks @lochana-chathura

**Comment by lochana-chathura** [comment]
Fixed in #34590.
